Modules/_pickle.c

```diff
@@ -1959,25 +1959,29 @@ whichmodule(PickleState *st, PyObject *global, PyObject *global_name, PyObject *
        extra parameters of __import__ to fix that. */
     module = PyImport_Import(module_name);
     if (module == NULL) {
+        Py_DECREF(module_name);
```

Comment:
There's a use of `module_name` in the error message below. Maybe move the `Py_DECREF()` after the `PyErr_Format()`?

Reply:
Indeed!